aviary/subsystems/mass/simple_mass/tail.py

```diff
@@ -0,0 +1,281 @@
+                       units='deg', 
+                       desc="Twist distribution")
+        
+        # Outputs
+        add_aviary_output(self,
+                            Aircraft.HorizontalTail.MASS,
+                            units='kg', 
+                            desc="Total mass of the tail")
+        #else:
+        add_aviary_output(self,
+                            Aircraft.VerticalTail.MASS,
+                            units='kg', 
+                            desc="Total mass of the tail")
+
+    def compute_primal(self, 
+                       aircraft__horizontal_tail__span, 
+                       aircraft__horizontal_tail__root_chord,
+                       aircraft__vertical_tail__span, 
+                       aircraft__vertical_tail__root_chord, 
+                       tip_chord_tail, 
+                       thickness_ratio, 
+                       skin_thickness, 
+                       twist_tail):
+        tail_type = self.options["tail_type"]
+        airfoil_type = self.options["airfoil_type"]
+        material = self.options['material']
+        density = materials.get_val(material, 'kg/m**3')
+        airfoil_file = self.options['airfoil_file']
+        num_sections = self.options['num_sections']
+        NACA_digits = self.options['NACA_digits']
+
+        # This is just so that the differentiation and unittest do not break. If tail_type = horizontal, ignore the vertical tail mass and vice versa.
+        # TODO: Potentially write these tails as separate files. 
+        aircraft__horizontal_tail__mass = 0.0 * thickness_ratio
+        aircraft__vertical_tail__mass = 0.0 * thickness_ratio
+
+        # File check
+        if airfoil_type == 'file':
+            if airfoil_type == 'file' and (airfoil_file is None or not os.path.isfile(airfoil_file)):
+                raise FileNotFoundError(f"Airfoil file '{airfoil_file}' not found or not provided.")
+            try: 
+                airfoil_data = np.loadtxt(airfoil_file, skiprows=1) # Assume a header
+                x_coords, y_coords = airfoil_data[:, 0], airfoil_data[:, 1]
+            except Exception as e:
+                raise ValueError(f"Error reading airfoil file: {e}")
+        
+        # Compute section airfoil geometry
+        if airfoil_file and os.path.exists(airfoil_file):
+            airfoil_data = np.loadtxt(airfoil_file)
+            x_coords = airfoil_data[:, 0]
+            y_coords = airfoil_data[:, 1]
+
+            camber, camber_location, max_thickness = self.extract_airfoil_features(x_coords, y_coords)
+        else:
+            # Parse the NACA airfoil type (4-digit)
+            camber = int(NACA_digits[0]) / 100.0 # Maximum camber
+            camber_location = int(NACA_digits[1]) / 10.0 # Location of max camber
+            max_thickness = int(NACA_digits[2:4]) / 100.0 # Max thickness
+        
+        # Tail type check
+        if tail_type not in ['horizontal', 'vertical']:
```

Comment:
You actually don't need this check because the OpenMDAO variable dictionary will raise an error if a user tries to set it to something that is not in `values`.